cmd/kogito/command/deploy/deploy_service.go

```diff
@@ -47,24 +47,25 @@ var (
 
 type deployFlags struct {
 	CommonFlags
-	name              string
-	runtime           string
-	serviceLabels     []string
-	incrementalBuild  bool
-	buildEnv          []string
-	reference         string
-	contextDir        string
-	source            string
-	imageS2I          string
-	imageRuntime      string
-	native            bool
-	buildLimits       []string
-	buildRequests     []string
-	imageVersion      string
-	mavenMirrorURL    string
-	enableIstio       bool
-	enablePersistence bool
-	enableEvents      bool
+	name                string
+	runtime             string
+	serviceLabels       []string
+	incrementalBuild    bool
+	buildEnv            []string
+	reference           string
+	contextDir          string
+	source              string
+	imageS2I            string
+	imageRuntime        string
+	native              bool
+	buildLimits         []string
+	buildRequests       []string
+	imageVersion        string
+	mavenMirrorURL      string
+	enableIstio         bool
+	enablePersistence   bool
+	enableEvents        bool
+	mavenDownloadOutput bool
```

Comment:
Could we have something like `enableMavenDownloadOutput` for the option as it is a boolean value ?

Reply:
changed name to enableMavenDownloadOutput